src/api.js

```diff
@@ -531,6 +535,11 @@
     bus.subscribe(this.getEventName(connect.ContactEvents.CONNECTED), f);
   };
 
+  Contact.prototype.onError = function (f) {
```

Comment:
We are adding new apis so we should bump the minor version, not the bug version.